client/src/Components/MyOffers/index.jsx

```diff
@@ -1,6 +1,82 @@
-import React from 'react';
-import UnderConstruction from '../UnderConstruction';
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import { Card, Row, Col, Container, Spinner } from 'react-bootstrap';
+import offersDetails from '../utilis/myOffers';
+import './style.css';
```

Comment:
It'd be good if you separate importing modules from local components and files or style